### DISCUSSION TITLE
Is there a way to write signature seed fields in the command client?

### DISCUSSION THREAD

**Discussion by regonzalom** [MatthiasValvekens/pyHanko#117] [General]
Hello,

I can create new hidden fields with the command line client, but I want to write reason and location seed fields. Is it possible?

Greetings,

**Comment by MatthiasValvekens** [comment]
Unfortunately not, or at least not yet. You'll have to write a small wrapper around the library to do that.

It wouldn't be especially hard to add to the CLI, but I'm hesitant to add more features to the CLI layer because it is already quite messy as-is (it grew pretty "organically" over the years), and I'd like the opportunity to clean it up before tacking on more stuff.
That said, if you're up for adding the functionality to the CLI yourself (e.g. in `addfields`), I'm open to pull requests.
